.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will run tests using node and then publish a
+# package to GitHub Packages when a release is created
+# For more information see:
+# https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm test
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+
+  publish-gpr:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Does this have any purpose? - I thought that GPR was for private packages really.

Reply:
To my knowledge it's just the same as NPM, but I could be wrong... my intention here was merely _not_ to constrain where someone wanted to pull the release.

Reply:
I'd suggest avoiding GPR for now. I'm pretty sure I read it's unfriendly to public packages. i.e. You need to log-in to pull.

I've also just read that your package must be scoped and snowtify is not (currently).

Reply:
Oooph, "_login to pull!?_" - that's very poor form!
What does `scoped` mean?

Reply:
```
"name": "@ukhomeoffice/snowtify"
```

Reply:
i.e. It can't just be called `snowtify`.